src/Display/Modal/ModalStyle.ts

```diff
@@ -87,7 +87,15 @@ export const ModalContentArea = styled.div<ModalContentAreaProps>`
       `;
     }
   }}
-    
+
+  @media (max-width: ${ScreenSize.tablet}px) {
```

Comment:
Currently
```
@media (max-width: ${ScreenSize.mobileM}px) {
    width: 95vw;
  }
```

This helps for the s and m modal sizes as their width is less than the mobileM and tablet device size.

For the sizes `l` and `xl` this criteria should be followed from tablet as the widths of the two are more than the tablet and mobileL device sizes.